js/app.ts

```diff
@@ -22,7 +22,7 @@ nunjucks.configure(['views/'], {
     noCache: true
 })
 
-app.use((error, req, res, next) => {
+app.use((error: any, req: Request, res: Response, next: NextFunction) => {
```

Comment:
I put "any" type for Error instead of "Error", because of exception "Property 'code' does not exist on type 'Error'".
What do you think?